### DISCUSSION TITLE
Beschriftung von Markern in dynamischen Inhalten

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#115] [Q&A]
Für die Beschriftung von Markern für Verkehrszählstellen (welche über dynamische Inhalte dargestellt werden), haben wir in der custom.js einen label definiert: 
var label = f.properties.ZST_Nummer.
Idee wäre die Zählstellennummer im Marker zu labeln. Allerdings ist mein JS Wissen eher rudimentär und so kommt es, dass es so aussieht wie im Screenshot. Die Zählstellennummer ist ein Code der aus 6 Stellen besteht (z.B: 9057_3)
![screenshot_zaehlstellen](https://github.com/gis-eni/webgis-forum/assets/100088983/0f84cdd9-d3f6-4af0-a91d-4920a55090ad)

Wie kann man das besser machen?

**Comment by gis-eni** [comment]
In den Markern haben 6 Zeichen keinen Platz. Das Maximum was noch geht sind 3 Zeichen. Also sowas wie Hausnummern, etc.
